docs/hacking/building-from-source.md

```diff
@@ -0,0 +1,27 @@
+# Building from Source
+
+## Prerequisites
+
+Before you begin, ensure you have the following software installed on your system:
```

Comment:
Let’s simply refer to the getting started documentation, for example: first, install the required system dependencies as specified on that page.

Reply:
Otherwise, we need to maintain the list of dependencies in two places.